### ISSUE TITLE
Manga.ae

### ISSUE THREAD

**Issue by Abdelilah91** [tachiyomiorg/tachiyomi-extensions#16201]
### Source name

مانجا العرب

### Source link

https://manga.ae

### Source language

العربية

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by rashid19961** [comment]
i have tried a fix and doesnt work , I recommend you to migraite to a different source since they have upped their cloudflare level and that triggers a cloudflare error

**Comment by Seishirou101** [comment]
manga.ae is currently linked to https://mngaar.com but it looks like that is down. From previous issues, it looks like mngaar was their site that had lower cloudflare protection. Not sure if mngaar is gone forever, but it possibly might just need a domain change...

**Comment by Abdelilah91** [comment]
yeah I think mangaar is gone and it won't work anymore, mangaae is their
new site.
Could you please add it as a new source instead of mangaar ?!

Le mar. 2 mai 2023 à 17:50, Seishirou101 ***@***.***> a
écrit :

**Comment by Abdelilah91** [comment]
Really, so there's no chance we can add it at all😭?
